### PR TITLE
Add setup for mongo.

### PR DESCRIPTION
**Pull request by kprakobkit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # To run the app, follow these steps:
 
 1. Run 'go setup'.
-2. Open two terminal windows.
-3. Run 'go client' in one of the windows, and 'go server' in the other.
-4. Open localhost:8081 and start developing
+2. Open three terminal windows.
```
Comment (achiu8) [inline README.md:4]: lol

File: models/order.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import mongoose from 'mongoose';
+
+var orderSchema = new mongoose.Schema({
+  status: String
```
Comment (achiu8) [inline models/order.js:4]: Do we need ID?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+3. Run one of the three commands in each window: 'go client', 'go server', and 'go mongo'.
```
Comment (achiu8) [inline README.md:5]: How do you get this to be `go [function]` instead of `./go [function]`?

File: README.md (hunk 1 above)
Comment (kprakobkit) [inline README.md:4]: yeah i'm thinking of just running mongo in the background...so many windows right now.

File: models/order.js (hunk 2 above)
Comment (kprakobkit) [inline models/order.js:4]: yes def i just made it as a place holder for now. will work on it some more tonight.

**Pull request merged**